openvdb_points/unittest/TestPointCount.cc

```diff
@@ -58,12 +58,12 @@ class TestPointCount: public CppUnit::TestCase
 
 }; // class TestPointCount
 
+using namespace openvdb;
```

Comment:
Is this necessary? Every unit test has it, and we don't seem to be making use of it in the declarations below...